sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_configuration.py

```diff
@@ -36,7 +42,7 @@ class LogsIngestionClientConfiguration(Configuration):  # pylint: disable=too-ma
 
     def __init__(self, endpoint: str, credential: "TokenCredential", **kwargs: Any) -> None:
         super(LogsIngestionClientConfiguration, self).__init__(**kwargs)
-        api_version = kwargs.pop("api_version", "2021-11-01-preview")  # type: str
+        api_version: Literal["2021-11-01-preview"] = kwargs.pop("api_version", "2021-11-01-preview")
```

Comment:
You might need to add typing-extensions to the setup.py if you haven't already

Reply:
Thanks for the reminder!